Storj.cyberduckprofile

```diff
@@ -0,0 +1,2558 @@
+<plist version="1.0">
+<dict>
+<key>Protocol</key>
+<string>s3</string>
+<key>Vendor</key>
+<string>Storj Labs</string>
+<key>Description</key>
+<string>Storj DCS</string>
+<key>Default Hostname</key>
+<string>gateway.storjshare.io</string>
+<key>Hostname Configurable</key>
+<true/>
```

Comment:
Are you sure you want to allow other endpoints than `gateway.storjshare.io`?

Reply:
I think so.  Its possible to connect directly to regional PoP endpoints such as https://gateway.us1.storjshare.io/.  While there is seldom any utility in doing so, it seems sensible to allow this.